### DISCUSSION TITLE
Any way to ignore index in display?

### DISCUSSION THREAD

**Discussion by mark-marinelli** [keirf/flashfloppy#595] [Q&A]
I'm using a couple of Goteks with an Amiga 500 and an IBM PS/1, both of which have OLED displays. In order to manage a large number of multi-disk programs, I'm using indexed file names (prefix "_") to keep disks grouped and ordered. But the indexes take up space in the OLED display, e.g. `_0000-This is a blank floppy.IMG`, and I'd love if I could ignore them and simply display `This is a blank floppy.IMG` instead, dropping the _0001. Is there already a setting for this that I'm missing in the FF.CFG file?

**Comment by keirf** [comment]
FlashFloppy sorts filenames so just put whatever short prefix you like if needed at all. Eg just 0 1 2...

**Reply by mark-marinelli** [reply]
Thanks for the quick reply. I know that the files will be sorted, was just wondering if the display could ignore the index which I'm only using for sorting, but which isn't actually relevant to the file name itself. I figured that since the indexing functionality is aware to treat an index prefix as a sort key, it may also be able to leverage that to exclude the index from display.

**Comment by keirf** [comment]
I'm saying don't use indexed mode. No need.
